app/models/concerns/benchmark_indicator_seed.rb

```diff
@@ -11,13 +11,19 @@ def seed_indicators!
       return unless BenchmarkIndicator.count.zero?
 
       warn "Seeding data for BenchmarkIndicators..."
-      benchmark_indicators_attrs = JSON.load File.open File.join Rails.root, "/db/seed-data/benchmark_indicators.json"
+      benchmark_indicators_attrs =
+        JSON.load File.open File.join Rails.root,
+                                      "/db/seed-data/benchmark_indicators.json"
       benchmark_indicators_attrs.each do |hash_attrs|
         attrs = hash_attrs.with_indifferent_access
-        technical_area = BenchmarkTechnicalArea.find_by_sequence! attrs[:technical_area_sequence]
+        technical_area =
+          BenchmarkTechnicalArea.find_by_sequence! attrs[
+                                                     :technical_area_sequence
+                                                   ]
```

Comment:
NIT: Oof this seems ugly to me 🤷‍♂️ Just saying. No change needed.

Reply:
No action needed.

Reply:
agreed, but also 🤷‍♂ - it's the tool's fault!